docs/source/usage/reference.rst

```diff
@@ -50,3 +50,240 @@ Thank you and enjoy PIConGPU and our community!
+ - PhD thesis: LWFA betatron radiation (mainly experiment) [Koehler2019]_
+ - PhD thesis: LWFA Coherent transistion radiation diagnostics (CTR) (mainly experiment) [Zarini2019]_
+ - PhD thesis: Laser ion acceleration (mainly experiment) [Obst-Huebl2019]_
+ - PhD thesis: Exascale simulations with PIConGPU, laser ion acceleration  [Huebl2019]_
+ - Bachelor thesis: Synthetic coherent transistion radiation [Castens2019]_ 
+
+
+
+
+List of references in chronological order
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+.. [Burau2010]
+       Burau, H., et al.,
+       *A fully relativistic particle-in-cell code for a GPU cluster.*,
+       IEEE Transactions on Plasma Science, 38(10 PART 2), 2831–2839 (2010),
+       https://doi.org/10.1109/TPS.2010.2064310
+
+.. [Zuehl2011]
+       Zühl, L.,
+       *Visualisierung von Laser-Plasma-Simulationen*,
+       Diploma Thesis at Technische Universität Dresden & Helmholtz-Zentrum Dresden - Rossendorf for the German Degree "Diplom-Informatiker" (2011),
+       https://www.hzdr.de/db/Cms?pOid=35687
+
+.. [Pausch2012]
+       Pausch, R.,
+       *Electromagnetic Radiation from Relativistic Electrons as Characteristic Signature of their Dynamics*,
+       Diploma Thesis at Technische Universität Dresden & Helmholtz-Zentrum Dresden - Rossendorf for the German Degree "Diplom-Physiker" (2012),
+       https://doi.org/10.5281/zenodo.843510
+
+.. [Bussmann2013]
+       Bussmann, M. et al.
+       *Radiative signatures of the relativistic Kelvin-Helmholtz instability*
+       SC ’13 Proceedings of the International Conference for High Performance Computing, Networking, Storage and Analysis (pp. 5-1-5–12)
+       https://doi.org/10.1145/2503210.2504564
+	
+.. [Huebl2014]
+        Huebl, A.,
+        *Injection Control for Electrons in Laser-Driven Plasma Wakes on the Femtosecond Time Scale*,
+        Diploma Thesis at Technische Universität Dresden & Helmholtz-Zentrum Dresden - Rossendorf for the German Degree "Diplom-Physiker" (2014),
+        https://doi.org/10.5281/zenodo.15924
+
+.. [Garten2015]
+        Garten, M.,
+	*Modellierung und Validierung von Feldionisation in parallelen Particle-in-Cell-Codes*,
+	Master Thesis at Technische Universität Dresden & Helmholtz-Zentrum Dresden - Rossendorf (2015)
+	https://doi.org/10.5281/zenodo.202500
+
+.. [Burau2016]
+        Burau, H.,
+	*Entwicklung und Überprüfung eines Photonenmodells für die Abstrahlung durch hochenergetische Elektronen*,
+	Diploma Thesis at Technische Universität Dresden & Helmholtz-Zentrum Dresden - Rossendorf for the German Degree "Diplom-Physiker" (2016),
+	https://doi.org/10.5281/zenodo.192116
+
+.. [Matthes2016]
+        Matthes, A.,
+	*In-Situ Visualisierung und Streaming von Plasmasimulationsdaten*,
+	Diploma Thesis at Technische Universität Dresden & Helmholtz-Zentrum Dresden - Rossendorf for the German Degree "Diplom-Informatiker" (2016),
+	https://doi.org/10.5281/zenodo.55329
+
```

Comment:
```suggestion

.. [Huebl2017]
         Huebl, A. et al.
	*On the Scalability of Data Reduction Techniques in Current and Upcoming HPC Systems from an Application Perspective*,
	ISC High Performance Workshops 2017, LNCS 10524, pp. 15-29 (2017),
	https://doi.org/10.1007/978-3-319-67630-2_2

```

Reply:
@ax3l Thanks for commenting this pull request. But this is a work in progress draft and I did not continue to work on this due to the issue of double references _ I will tray your solution above and then will add further references - I would never forget your plasma-p*** publication.